Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -309,4 +312,244 @@ theorem psum_isSymmetric (n : ℕ) : IsSymmetric (psum σ R n) := rename_psum _
 
 end PowerSum
 
+section Exponent
+
+open Classical
+open Composition
+open Finset
+
+variable (σ R) [CommSemiring R] [CommSemiring S] [Fintype σ] [Fintype τ]
+
+@[ext]
+structure Exponent (n : ℕ) :=
```

Comment:
We surely want a docstring here (and see any other mathlib file for the header of a file).